wire-ios-sync-engine/Tests/Source/Integration/BackgroundTests.m

```diff
@@ -67,26 +67,34 @@ - (void)testThatItSendsUILocalNotificationsForExpiredMessageRequestsWhenGoingToT
     [self.userSession performChanges:^{
         [conversation appendMessageWithText:@"foo"];
     }];
-    WaitForAllGroupsToBeEmpty(0.5);
     
     // background
     [self.application simulateApplicationDidEnterBackground];
     [self.application setBackground];
-    
+
+    self.mockTransportSession.responseGeneratorBlock = ^ZMTransportResponse *(ZMTransportRequest *request) {
+        (void)request;
+        NSError *requestExpiredError = [NSError errorWithDomain:ZMTransportSessionErrorDomain code:ZMTransportSessionErrorCodeRequestExpired userInfo:nil];
+        return [ZMTransportResponse responseWithTransportSessionError:requestExpiredError apiVersion:0];
+    };
     [self.mockTransportSession expireAllBlockedRequests];
     WaitForAllGroupsToBeEmpty(0.5);
 
     // then
     XCTAssertEqual(self.notificationCenter.scheduledRequests.count, 1u);
 }
 
+// TODO: jacob doesn't really make sense anymore
```

Comment:
you mean the test?

Reply:
Yeah but then I ended up fixing the test anyway but I forgot to remove this comment.

Reply:
Removed the comment